http-client/ChangeLog.md

```diff
@@ -1,3 +1,8 @@
+## 0.5.12.2
```

Comment:
Since this is adding a new export, the version should be 0.5.13.

Reply:
:+1: fixed